tools/nvdvuln/nvdvuln.go

```diff
@@ -70,6 +70,9 @@ func main() {
 	var softwareCPEs []fleet.SoftwareCPE
 	ds.UpsertSoftwareCPEsFunc = func(ctx context.Context, cpes []fleet.SoftwareCPE) (int64, error) {
 		softwareCPEs = cpes
+		for _, cpe := range cpes {
+			fmt.Printf("Matched CPE: %s\n", cpe.CPE)
```

Comment:
I know this was merged, but wanted to know:
Where does this `fmt.Printf` ending up being printed? stdout? other?